ellis.yaml

```diff
@@ -172,6 +174,7 @@ resources:
             public_ip=__public_mgmt_ip__
             public_hostname=ellis-__index__.__zone__
             etcd_cluster=$etcd_ip
+            xdms_hostname=homer-__index__.__zone__:7888
```

Comment:
I'd go for homer-0 explicitly (in theory, you could have 2 Ellises and one Homer, in which case the second Ellis would try and contact homer-1 which doesn't exist).